### DISCUSSION TITLE
Decorative Rugs

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#1611] [Art]
We are looking for some variations of Rugs to add to Sunflower Land.

Rugs are the perfect accessory if you want to show off a rare item on your land. Players will be able to place these under rare items and their Bumpkins to make them look even more special!

![land-export](https://user-images.githubusercontent.com/11745561/199653049-2c73f784-fa12-4bc8-a1cc-69737a6163c5.png)

We are looking for ideas and designs for a range of different Sunflower Land inspired Rugs! Feel free to do set submissions or one-off ideas ⭐

## Submissions

- Fits game aesthetic and maintains same dimensions
- Original pieces of work
- Pixel sizing is 1x1px
- Dimensions 28px wide and 28px height

_By contributing to the project you agree to the terms and conditions - https://docs.sunflower-land.com/support/terms-of-service. Your art can be used freely by the project and all community members_

When submitting art, please include the following:

- Your farm ID (we will make sure the artists get some goodies 🏆 )
- A minified version of your art piece
- An example of your rug on the land

![land](https://user-images.githubusercontent.com/11745561/199652971-830c85ed-4993-4ba7-99ec-9e55504bd4cd.png)
![land](https://user-images.githubusercontent.com/11745561/199652933-f086590b-5d57-49fc-98c6-4c201d904c68.png)
![rugs](https://user-images.githubusercontent.com/11745561/199652935-fd853a58-d24b-46c6-9c48-61573ff97fcd.png)

**Timeline**

We will be looking to introduce the initial collectible rugs into the game in 3-4 weeks and would love to showcase some community art ❤️

As always, contributors will receive an airdrop of the game item they design (if it is tokenised) and also some goodies!

Winners will receive an airdrop of their rug + a care package of SFL & gold 🏆

**Comment by MrSnorch** [comment]
Farm ID: 14106

![коверv1](https://user-images.githubusercontent.com/104712725/199849725-1a1c11ce-be68-4f58-9425-30ea85f46c5d.png)
![ковер2-export](https://user-images.githubusercontent.com/104712725/199849792-03c080f7-b386-4fb1-b1ad-1b351f74b10b.png)

![rug v2](https://user-images.githubusercontent.com/104712725/201090404-4d04f012-5cc0-4a0c-b29c-d86b3a858001.png)
![rur v2 prew](https://user-images.githubusercontent.com/104712725/201090415-ac8943f3-0afd-4524-a89e-846b79053b11.png)

Old Rug
![old rug](https://user-images.githubusercontent.com/104712725/201090742-aa4ec9d1-68a9-4102-8a59-826a66e9ced5.png)
![old rug prew](https://user-images.githubusercontent.com/104712725/201090751-939e98f4-c957-4fa6-b9f5-e266e2e1e6e4.png)

**Comment by Attalante** [comment]
![Rug](https://user-images.githubusercontent.com/103903393/199862935-34851c31-5f9e-415c-9980-198f011aae1d.png)
![199652971-830c85ed-4993-4ba7-99ec-9e55504bd4cd](https://user-images.githubusercontent.com/103903393/199862951-1c00f77e-1598-482f-8779-974e6b569c2a.png)

Farm ID: 245
(First try in pixel art, feel free to give reviews !)

**Comment by Designforu** [comment]
Farm id: 134612


![rug1](https://user-images.githubusercontent.com/114425639/199955520-3d9495e9-9b9b-4124-af0e-5080733c2daa.png)
![rug1 ](https://user-images.githubusercontent.com/114425639/199955595-47c1180d-8dd8-45dc-9d71-24ebdbae9c4b.png)

![rug2](https://user-images.githubusercontent.com/114425639/199955784-b1df5405-b05d-4247-8468-a479e1bcc6c2.png)
![rug2](https://user-images.githubusercontent.com/114425639/199955799-35be9896-a229-4a0c-ae70-c0806392d455.png)

![rug3](https://user-images.githubusercontent.com/114425639/199955823-e953467a-c522-48a4-8f6d-9753f6fcf3b0.png)
![rug3](https://user-images.githubusercontent.com/114425639/199955839-fba53171-3236-40d9-960b-251e461045fe.png)

![rug4](https://user-images.githubusercontent.com/114425639/199955891-d613beb3-50f1-41bc-a073-7b0be0eb2c52.png)
![rug4](https://user-images.githubusercontent.com/114425639/199955905-415ab47c-5524-4777-82ec-105c610640db.png)

![rug5](https://user-images.githubusercontent.com/114425639/200341141-3d1790cc-ddb9-4409-8391-92b336a2fb83.png)
![rug5](https://user-images.githubusercontent.com/114425639/200341166-70aa8923-b9dc-4d60-84a5-49e05b2707bf.png)

![rug6](https://user-images.githubusercontent.com/114425639/200341216-b7202bef-2a8e-49bf-adcc-cda3c745d280.png)
![rug6](https://user-images.githubusercontent.com/114425639/200341234-f0d31d22-6d9a-4021-a5b5-4e1560bf53c9.png)

**Comment by JcEii** [comment]
Farm Id: 15747

### **Pattern Rugs (Philippine Pattern Rugs)**
**Sample:**
![asset](https://user-images.githubusercontent.com/105348036/200014893-d1cbe159-f425-4216-8a41-65719312219c.gif)

**Minified:**
![philippine pattern rug1](https://user-images.githubusercontent.com/105348036/200014837-a358c06c-4562-4312-b1a0-c8330e4ee988.png)
![philippine pattern rug2](https://user-images.githubusercontent.com/105348036/200014843-857e02c9-6792-469c-b5c3-474b19bebbb2.png)
![philippine pattern rug3](https://user-images.githubusercontent.com/105348036/200014845-a49997ca-f291-4f57-bc0d-e52f64a8a30e.png)
![philippine pattern rug4](https://user-images.githubusercontent.com/105348036/200014848-e9e3af8e-8dcc-4542-920b-413cf6a0ca93.png)
![philippine pattern rug5](https://user-images.githubusercontent.com/105348036/200014851-9745114a-4179-4e2c-b9d4-895bd530ad72.png)
![philippine pattern rug6](https://user-images.githubusercontent.com/105348036/200014855-c08f11a4-8374-4213-b13c-b00bbaaeede4.png)
![philippine pattern rug7](https://user-images.githubusercontent.com/105348036/200014859-956c907f-16fe-4379-b5e2-873bfb5144ed.png)
![philippine pattern rug8](https://user-images.githubusercontent.com/105348036/200014862-785b17cf-8232-417e-9826-26317b26db57.png)
![philippine pattern rug9](https://user-images.githubusercontent.com/105348036/200014863-dcc6e03a-2972-42a4-a9f7-4b313bb86c0c.png)

**Comment by esp123** [comment]
Sacul Tibute Rug
![sacul](https://user-images.githubusercontent.com/74337946/200101415-a7da3348-86d7-4557-8da4-9c5253af4b2c.png)

Popcorn Rugs
![bright rwd](https://user-images.githubusercontent.com/74337946/200101426-77365c3a-fa2a-4355-a51a-9de7089d2314.png)
![red](https://user-images.githubusercontent.com/74337946/200101430-86c88aff-9437-45a9-9f51-a13847a7d029.png)
![brown rug movie](https://user-images.githubusercontent.com/74337946/200101431-14a3232f-6ffc-4705-aec5-27c0e8ce7bf8.png)

**Comment by LittleEins** [comment]
### Little Eins Rugs 
Farm ID: 73795
### Summoning Rugs
![rugs](https://user-images.githubusercontent.com/104184348/200117939-f0def5fc-efdc-4ef7-b0c8-4563ff72470c.png)
![summoning rugs](https://user-images.githubusercontent.com/104184348/200117934-6849cc80-c0e5-4bec-83d6-c0fa1e4930c6.png)
### Royalty Black Rug
![rugs1](https://user-images.githubusercontent.com/104184348/200603916-5775e82c-cbf2-4d75-99f3-d59d72ade79d.png)
![edgy-rugbig](https://user-images.githubusercontent.com/104184348/200603689-4b018d6b-0da8-4d53-8eff-b273913a84cd.png)
![edgy-rug](https://user-images.githubusercontent.com/104184348/200603758-76c50283-5ed4-40c1-82f2-0c60a08fd8f5.png)
### Rock Rug
![rugs2](https://user-images.githubusercontent.com/104184348/200613368-bf7a5aa1-9900-4d9e-b0b8-1013a7469679.png)
![rock-rug](https://user-images.githubusercontent.com/104184348/200613395-d07db17f-15c3-460a-9889-12e0ce6cf293.png)
### Leaf Rug
![rugs3](https://user-images.githubusercontent.com/104184348/200629677-dfd5345f-bebd-4497-957f-90bb5aa5e4d1.png)
![eco-rug](https://user-images.githubusercontent.com/104184348/200629699-d6118826-217c-4afd-bc23-42a11bd08380.png)
### Honeycomb Rugs
![rugs5](https://user-images.githubusercontent.com/104184348/201511540-8d23a372-622f-40ec-9eca-5739761e386d.png)
![rugs4](https://user-images.githubusercontent.com/104184348/201511531-ccf52c31-1443-4d5f-83e8-733c4a3b5da3.png)
![honey-comb-rug2](https://user-images.githubusercontent.com/104184348/201511545-bc4a8247-e953-4bdd-a13d-001d565addf7.png)
![honey-comb-rug](https://user-images.githubusercontent.com/104184348/201511550-2d34e5ea-eba4-460f-8896-01d49f65f1de.png)

**Comment by silmeria18** [comment]
![royal rug](https://user-images.githubusercontent.com/108312168/200126293-e7cb9226-e0ad-4866-b3df-f0d36c2ba70c.png)
![Untitled 11-05-2022 02-00-04](https://user-images.githubusercontent.com/108312168/200126306-ca9eb55a-1ab8-485a-984b-c206ec898f04.png)
![rug on land](https://user-images.githubusercontent.com/108312168/200126309-d5756fb0-5f4c-4cf5-98dd-93cdd32bc776.png)
Farm id 11650
Royal Rug

**Comment by On1el** [comment]
Carpet for Confirmed Twitter and Sunflower World accounts
![Ковры галочкиХ5](https://user-images.githubusercontent.com/104461938/200181553-ce061ed5-30ed-41db-8017-b4df606a6d68.png)
![Ковры галочки](https://user-images.githubusercontent.com/104461938/200181565-16ec5f6a-3f70-43ef-bc99-6ec02b496755.png)
![галочки](https://user-images.githubusercontent.com/104461938/200181567-3f691856-b9bb-49d7-92de-0c7b8dfad05c.png)
![галочки2](https://user-images.githubusercontent.com/104461938/200181569-de117f4f-dde4-43a6-b63b-f1d2d8aab746.png)

**Comment by FarmerKuenster** [comment]
![rug options on land](https://user-images.githubusercontent.com/104322749/200210730-04ff01f4-1bf5-44d7-8b3d-4f333e60db0c.gif)
![rug1](https://user-images.githubusercontent.com/104322749/200210672-5e7140dc-e58b-40ed-aa47-ba75197e836c.png)
![rug2](https://user-images.githubusercontent.com/104322749/200210688-2a78d762-0949-458d-b85b-7c245220656b.png)
![rug3](https://user-images.githubusercontent.com/104322749/200210699-c6676054-0790-4586-93a9-95b21ff381eb.png)

discord: farmerkunester#1626
land ID: #32391

**Comment by tryjapa** [comment]
**Farm ID: 315**

Crop rugs;

**Preview:**
![crops-rugs](https://user-images.githubusercontent.com/9723813/197309076-1d2580f2-5583-4930-bd12-ec06d4caedfc.gif)


**Minified:**
Sunflower rug:
![sunflower-hug](https://user-images.githubusercontent.com/9723813/197309245-005898db-bca0-412b-95e2-218a3124431e.png)
Cauliflower rug:
![cauliflower-hug](https://user-images.githubusercontent.com/9723813/197309249-08d7f0b0-000a-4adc-8e67-31eb8f288a5a.png)
Pumpking rug:
![pumpkin-hug](https://user-images.githubusercontent.com/9723813/197309257-d71b556b-334b-4daa-bcaf-457f2cebb671.png)
Cabbage rug:
![cabagge-hug](https://user-images.githubusercontent.com/9723813/197309260-331f6e6d-35b7-40c1-8f54-1571457b23ea.png)

**Comment by shinooon** [comment]
Preview
![sunflower rug preview](https://user-images.githubusercontent.com/109294090/200481675-5a551440-a71a-4506-869b-b21566f2ef99.png)
![sunflower rug preview](https://user-images.githubusercontent.com/109294090/200481409-4f4305fd-903b-4b14-9509-e837e478e6db.png)
Sunflower Rug
![sunflower rug](https://user-images.githubusercontent.com/109294090/200481452-32e175ce-9683-44c2-975c-69aadf8ba97a.png)


Farm ID: 51738

**Comment by VentinRD** [comment]
Farm ID: 10350
Discord: Ventin#5150

Anchor Rug
![Anchor Rug1](https://user-images.githubusercontent.com/103960383/200782183-e71164cf-9089-4bd1-8a78-68333629a10f.png)

World Map Rug
![World Map](https://user-images.githubusercontent.com/103960383/200782221-becd3e4a-1301-43ee-876f-ca48b8b58ced.png)

Ripped World Map Rug
![Ripped World Map](https://user-images.githubusercontent.com/103960383/200782250-e87791de-b118-42d3-8304-d73c3699e8f9.png)

![Rugs Subm](https://user-images.githubusercontent.com/103960383/200782663-41026bb6-02e1-4ba1-a9ec-29936688d69f.png)

**Comment by digital-farms** [comment]
# Ukrainian embroidered rug (Vyshyvanka)
### Farm ID: 21637
![vyshyvanka_rug_example](https://user-images.githubusercontent.com/105886943/200884175-b98b602b-e884-474e-ae66-1f41fc6c9aaf.png)
![vyshyvanka_rug](https://user-images.githubusercontent.com/105886943/200884181-ce3f6c4d-c3bb-46d6-8651-4d7e360074e9.png)

**Comment by tuzukurugoblin** [comment]
**FARM ID:10180**
![ezgif com-gif-maker](https://user-images.githubusercontent.com/109798853/201113272-0e233c36-253b-40a1-ad9c-610616c86b79.gif)
![mavi çerçeveli 1](https://user-images.githubusercontent.com/109798853/201113839-cd5660d5-a06c-4ae9-b327-2292753348cd.png)
![gf1](https://user-images.githubusercontent.com/109798853/201113851-e0dbc140-0a05-4844-b5c0-b017b9bf9653.png)
![yeşil kare 1](https://user-images.githubusercontent.com/109798853/201113936-a5a12c38-335f-4b94-ba60-f21044c7ab21.png)
![1 kare](https://user-images.githubusercontent.com/109798853/201113980-8244bd86-08af-4c9c-9a57-4a5787256af1.png)
![yeşil kalp 1](https://user-images.githubusercontent.com/109798853/201114010-8f4f6f72-c4c1-44d7-880f-c78237eac47e.png)
![yeşil 1](https://user-images.githubusercontent.com/109798853/201115415-a20e8965-df1b-4785-ac5f-a7199300d7a7.png)
![sarı kalpli 1](https://user-images.githubusercontent.com/109798853/201115479-22ef2be7-a4f3-4961-85eb-360f4e1c182a.png)
![sarı 1](https://user-images.githubusercontent.com/109798853/201115777-89aa075d-b2b2-42d3-92d9-7df5846ee3b9.png)
![yıldız deneme 1](https://user-images.githubusercontent.com/109798853/201115574-29de3277-705a-47e9-b2d8-ff8e88c06787.png)
![11](https://user-images.githubusercontent.com/109798853/201115608-3aad43b3-4b9f-49f3-9464-0ae0f09ca8fe.png)
![siyah beyaz 1](https://user-images.githubusercontent.com/109798853/201115633-880ebeed-37f2-4d56-a514-c4e221493d29.png)
![siyah beyaz](https://user-images.githubusercontent.com/109798853/201115652-35dba11e-d3c7-4c14-8b28-d04fda602cda.png)

**Comment by joaojuniorpt** [comment]
_Juninkill_
**ID farm: 138257
ID discord: juninkill#1620**

**_I made these arts to honor the Artists, Holders, coder and veteran players._**
![allsrugjr002](https://user-images.githubusercontent.com/20995533/201227649-5395db36-4835-4ade-953e-aea59251c4a8.gif)


![allsrugjr001](https://user-images.githubusercontent.com/20995533/201221160-ed0316c4-eb7a-4117-b702-44244900d239.gif)

**Artist**
![art01](https://user-images.githubusercontent.com/20995533/201222574-059884dc-5940-4955-9df4-e313bc546149.png)
![art](https://user-images.githubusercontent.com/20995533/201223087-7a5c2d8d-a9a3-446b-8494-c96130933b1e.png)

**holders**
![rugholder01](https://user-images.githubusercontent.com/20995533/201222623-5b23f7b5-ec88-4367-a136-f794edbffcc7.png)
![rugholder](https://user-images.githubusercontent.com/20995533/201223113-c52d9ff8-4a37-43b2-af34-300f25fb5a25.png)

**coder**
![rugcodes01](https://user-images.githubusercontent.com/20995533/201222631-cb440c32-e22b-4bb1-bbe0-f84e149cd001.png)
![rugcodes](https://user-images.githubusercontent.com/20995533/201223131-f326618f-c60d-4356-85e3-984932b404a7.png)

**veteran players**
![players01](https://user-images.githubusercontent.com/20995533/201222666-32e4590d-a362-4e3b-958d-3ea5faef60eb.png)
![players](https://user-images.githubusercontent.com/20995533/201223154-2223b9ee-138d-419a-aa89-a18690c6f0db.png)

**Comment by Galphath** [comment]
Farm ID: 170

Summoning sunflowers

![Rug1](https://user-images.githubusercontent.com/20891096/201264229-715d82ad-49e8-48a9-918e-bc7ecf3ead4a.png)
![MockupRug1](https://user-images.githubusercontent.com/20891096/201264273-52fcc612-2cc0-4bcd-ab54-73fceeb875ba.png)

Not exactly a Rug per se but an useful way to highlight an item XD

![Candles](https://user-images.githubusercontent.com/20891096/201264317-43e741e2-08ed-4fe9-89d1-3022a5574d23.gif)
![Candlesfloor](https://user-images.githubusercontent.com/20891096/201264324-cac29146-a449-4645-937c-0e2831d1d493.gif)

**Comment by peperol** [comment]
farm ID 145849

round red rug
![round red rug](https://user-images.githubusercontent.com/114354077/201903768-06ba08ba-03f2-4dbd-b569-3b886a927eb4.png)
![prova tondorug3red](https://user-images.githubusercontent.com/114354077/201903786-e611d1c8-10ba-4486-934d-813347e0d985.png)

sunflower round rug
![sunruground](https://user-images.githubusercontent.com/114354077/201903927-89ebc060-599d-47c1-b2b5-d3efd3bc2f33.png)
![prova tondorugsun](https://user-images.githubusercontent.com/114354077/201903949-f2329c01-0d12-407a-8228-898bab6a9982.png)

rug patchwork

![rug patchwork](https://user-images.githubusercontent.com/114354077/201982255-23527813-f82d-4bd9-b14f-3bc490378f95.png)
![rugpatchw](https://user-images.githubusercontent.com/114354077/201982283-98b2f89a-dd0f-4037-9e72-85390a80f950.png)

Noel rug

![1noelrug](https://user-images.githubusercontent.com/114354077/202140333-4b2364c3-72db-4b7d-8a46-db5a4506e711.png)

![noel rug](https://user-images.githubusercontent.com/114354077/202139913-68e87ed4-7658-4fb2-b6ed-48b20bef6691.png)

picnic rug

![picnicrug1](https://user-images.githubusercontent.com/114354077/202196465-26b35f32-22ba-4428-8053-4308fe619827.png)

![picnicrug2](https://user-images.githubusercontent.com/114354077/202196232-c80beb16-df44-4127-b6bd-ea518d17300f.png)


Fame rug

![fame rugred](https://user-images.githubusercontent.com/114354077/202283139-282b1894-61d9-4f88-a70c-d2d18650f75c.png)
![fame rugred2](https://user-images.githubusercontent.com/114354077/202283171-c8fca544-c059-4307-8594-83e26e481a0f.png)

**Comment by ghost** [comment]
**Fried Egg Rug** 

Preview: 
![image](https://user-images.githubusercontent.com/114235416/202924776-1baaa2b2-3722-41d5-9b61-889b85aa3189.png)

Minified: 
![image](https://user-images.githubusercontent.com/114235416/202924800-cceece6c-326d-4fe7-a398-0e3e34a0bd2d.png)

Farm ID: 58821
